hail_search/test_search.py

```diff
@@ -189,6 +196,16 @@
 
 OMIM_SORT_METADATA = ['ENSG00000177000', 'ENSG00000097046', 'ENSG00000275023']
 
+MULTI_PROJECT_BOTH_SAMPLE_TYPE_VARIANTS = [
+    deepcopy(v) for v in
+    [PROJECT_2_VARIANT, MULTI_PROJECT_VARIANT1, MULTI_PROJECT_VARIANT2, VARIANT3, VARIANT4]
+]
+for v in MULTI_PROJECT_BOTH_SAMPLE_TYPE_VARIANTS:
+    for indiv_id, gts in v['genotypes'].items():
+        v['genotypes'][indiv_id] = [gts]
+    if 'I000015_na20885' in v['genotypes']:
+        v['genotypes']['I000015_na20885'].append({**v['genotypes']['I000015_na20885'][0], 'sampleType': 'WGS'})
```

Comment:
rather than having an if statement in the loop, it would be better to explicitly use `PROJECT_2_VARIANT_BOTH_SAMPLE_TYPES` 
```suggestion
MULTI_PROJECT_BOTH_SAMPLE_TYPE_VARIANTS = [PROJECT_2_VARIANT_BOTH_SAMPLE_TYPES] + [
    deepcopy(v) for v in
    [MULTI_PROJECT_VARIANT1, MULTI_PROJECT_VARIANT2, VARIANT3, VARIANT4]
]
for v in MULTI_PROJECT_BOTH_SAMPLE_TYPE_VARIANTS[1:]:
    for indiv_id, gts in v['genotypes'].items():
        v['genotypes'][indiv_id] = [gts]
```

Reply:
this didn't quite work, but I added a second loop through `MULTI_PROJECT_BOTH_SAMPLE_TYPE_VARIANTS[:-2]` (because variants 3 and 4 don't have gts for that individual) to add the WGS gt

Reply:
I realized that variants 3 and 4 probably shouldn't have genotype arrays for individuals, but since `_has_both_sample_types` gets set for an entire request, as the code is written right now there will be an array with a single element.

I remember trying to conditionally return a genotype array or dictionary based on # of unique sampleTypes there were for an individual, but I was running into hail's rule of conditional expressions that the return value must be the same type.

Reply:
yeah I think it will be okay to have everything returned as an array instead of a dict to get the hail type checking to pass here. We just need to remember in the UI that the presence of a list does not always guarantee multiple genotypes